rest-api/README.md

```diff
@@ -0,0 +1,329 @@
+
+* Simple Example: `?simple=field`
+* Same Field; Multiple Values: `field=value&field=anotherValue`
+
+## Pagination
+Pagination is supported using the following query params:
+
+| Param          | Description                                                                 | Default |
+| -------------- | --------------------------------------------------------------------------- | ------- |
+| `page[offset]` | Specifies the starting position from where the records should be retrieved. | `0`     |
+| `page[limit]`  | Specifies the maximum number of records to be retrieved.                    | `10`    |
+
+### Example
+`/?page[offset]=0&page[limit]=10`
+
+---
+
+# Idempotency
+The IDs of individual tbDEX messages are used as idempotency keys
+
+---
+
+# Offerings
+
+The [`Offering`](../README.md#offering) resource is used to convey the currency pairs a PFI is _offering_. It includes information about payment methods and associated fees.
+
+## List Offerings
+
+### Description
+Used to fetch offerings from a PFI
+
+
+### Endpoint
+`GET /offerings`
+
+### Authentication
+No authentication required.
+
+### Authorization
+No authorization required. Offerings are publicly accessible
+
+### Query Params
+| Param              | Description                                                                                                                                                        |
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `baseCurrency`     | The primary currency listed in a currency pair, representing the unit being traded. **This is the currency that Alice will be _receiving_**                        |
+| `quoteCurrency`    | The secondary currency listed in a currency pair, indicating its value in relation to the base currency. **This is the currency that the PFI will be _receiving_** |
+| `payinMethodKind`  | The payin method Alice wishes to use to provide quote currency                                                                                                     |
+| `payoutMethodKind` | The payout method Alice wishes to use to receive base currency                                                                                                     |
+| `id`               | Query for a specific offering                                                                                                                                      |
+
+
+### Response
+| Status             | Body                  |
+| ------------------ | --------------------- |
+| `200: OK`          | `Offering[]`          |
+| `400: Bad Request` | `{ errors: Error[] }` |
+
+---
+
+# Threads
+A thread is a series of linked tbDEX messages between Alice and a PFI for a single exchange. A thread can be created by submitting an RFQ.
+
+## Submit RFQ
+
+### Endpoint
+`POST /threads/:thread_id/rfq`
```

Comment:
a couple questions -

1. a tbdex exchange always starts with an rfq. thoughts on calling this endpoint `POST /exchanges` and passing in the exchangeId (part of rfq) in the req body? this thought is similar to what @KendallWeihe proposed below, of using `PUT /threads/:thread_id` instead of `POST /threads/:thread_id/order`

2. are we aligning on using `exchanges` in place of `threads`? if so, it would be good to reflect that in submit rfq / order / close endpoint descriptions here and below.

Reply:
will surface this in an issue